catalog/video/video-admin-plugin/src/main/webapp/css/index.css

```diff
@@ -10,11 +10,27 @@
  *
  **/
 
+.td-name {
+    width: 20%;
+}
+
 .td-btn {
     border: 1px solid transparent;
     border-radius: 4px;
 }
 
+table {
+    table-layout: fixed;
```

Comment:
I think it'd be better to leave out this `table` block of styling as well as the `.td-name` styling.

Instead, I would add a style that applies to the `main` element.

```
main {
   overflow: auto;
}
```

This combined with the `max-width` on the `td` elements should result in a view that will accommodate extremes fairly well.